src/applications/personalization/profile360/vet360/tests/selectors.unit.spec.js

```diff
@@ -184,3 +186,53 @@ describe('selectors', () => {
   });
 
 });
+
+describe('selectVet360InitializationStatus', () => {
+  const old = { document: global.document };
+
+  beforeEach(() => {
+    hooks.beforeEach();
+    global.document = {
+      location: {
+        hostname: 'staging.vets.gov'
+      }
+    };
+  });
+
+  afterEach(() => {
+    global.document = old.document;
+  });
+
+  it('returns UNINITIALIZED if Vet360 is not found in the services array and there is not an associated transaction', () => {
+    state.user.profile.services = [];
+    const result = selectors.selectVet360InitializationStatus(state);
+    expect(result.status).to.be.equal(VET360_INITIALIZATION_STATUS.UNINITALIZED);
+  });
+
+  it('returns INITIALIZED if Vet360 is found in the services array', () => {
+    const result = selectors.selectVet360InitializationStatus(state);
+    expect(result.status).to.be.equal(VET360_INITIALIZATION_STATUS.INITIALIZED);
+  });
+
+  it('returns INITIALIZING if there is an ongoing transaction', () => {
+    const transactionId = 'transaction_1';
+    state.user.profile.services = [];
+    state.vet360.transactions = [
+      { data: { attributes: { transactionId, transactionStatus: TRANSACTION_STATUS.RECEIVED } } }
+    ];
+    state.vet360.fieldTransactionMap[INIT_VET360_ID] = { transactionId };
+    const result = selectors.selectVet360InitializationStatus(state);
+    expect(result.status).to.be.equal(VET360_INITIALIZATION_STATUS.INITIALIZING);
+  });
+
+  it('returns INITIALIZATION_FAILURE if there is a failed transaction', () => {
+    const transactionId = 'transaction_1';
+    state.user.profile.services = [];
+    state.vet360.transactions = [
+      { data: { attributes: { transactionId, transactionStatus: TRANSACTION_STATUS.REJECTED } } }
+    ];
+    state.vet360.fieldTransactionMap[INIT_VET360_ID] = { transactionId };
+    const result = selectors.selectVet360InitializationStatus(state);
+    expect(result.status).to.be.equal(VET360_INITIALIZATION_STATUS.INITIALIZATION_FAILURE);
+  });
+});
```

Comment:
Since the container is driven almost entirely by the selector, I just wrote test cases for the selector instead.